File: app/src/main/java/kotleni/b0mb3r/services/AtPrime.java
package kotleni.b0mb3r.services;

import okhttp3.FormBody;

public class AtPrime extends FormService {

    public AtPrime() {
        setUrl("https://api-prime.anytime.global/api/v2/auth/sendVerificationCode");
        setMethod(POST);
    }

    @Override
    public void buildBody(FormBody.Builder builder) {
        builder.add("phone", getFormattedPhone());
    }
}

File: app/src/main/java/kotleni/b0mb3r/services/AtPrime.kt
package kotleni.b0mb3r.services

import okhttp3.FormBody.Builder
import kotleni.b0mb3r.services.FormService
import kotleni.b0mb3r.services.SimpleBaseService

class AtPrime : FormService() {
    override fun buildBody(builder: Builder) {
        builder.add("phone", formattedPhone)
    }

    init {
        setUrl("https://api-prime.anytime.global/api/v2/auth/sendVerificationCode")
        setMethod(POST)
    }
}